### PR TITLE
Update functionality.html.erb.md

### PR DESCRIPTION
**Pull request by nitedog**
For **editor's discretion** as usual :)

### CODE REVIEW

File: source/carousels/functionality.html.erb.md [hunk 1]
```diff
@@ -598,9 +598,9 @@ In the example below, the list with buttons is added to the carousel using JavaS
 
 ### Focus the current carousel item
 
-Whe a user selects an item through the carousel navigation it should be receive focus immediately. This allows keyboard or assistive technology users to interact with the item immediately.
+When users select items through the carousel navigation buttons, the items should receive focus immediately. This makes interaction easier for keyboard and assistive technology users.
 
-By default, `<li>` itself cannot receive focus by default. However if the `tabindex` attribute is set to `-1`, JavaScript is able to set the focus to the items.
+By default, `<li>` elements cannot receive focus. Setting the `tabindex` attribute of the element to `-1` allows it to receive focus through JavaScript.
 
 To allow users to skip multiple items using the previous and next buttons, this approach is not used there.
```
Comment (nitedog) [inline source/carousels/functionality.html.erb.md]: Consider if we really need "putting it all together" - we have the full example anyway. Not sure, though - may be good to see the different stages of development? Is that useful for developers?

Comment (yatil) [inline source/carousels/functionality.html.erb.md]: Yeah, we didn’t have the full example (as prominent) previously – and I did not plan the tutorial that way. I planned this to be a step by step learning process, not a we do this and that an you can figure out what the change actually is implemented in the full example. To be honest, I’m at the moment thinking we should maybe start with the source code and then go along the source code and explain the rationale. But that would be too technical for our non-technical audience (which probably won’t look at the full source code anyway). I think I need to sleep over this for a night and then see :-)

File: source/carousels/functionality.html.erb.md [hunk 2]
```diff
@@ -598,9 +598,9 @@ In the example below, the list with buttons is added to the carousel using JavaS
...
```
Comment (yatil) [inline source/carousels/functionality.html.erb.md]: Nah, you’re right. The last sentence lacks coherence with the previous text.

File: source/carousels/functionality.html.erb.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ wcag_success_criteria:
   - 4.1.2
 ---
 
-Carousels display multiple content items one at a time, allowing users to browse through all of them in turn.
+Provide functionality to display the carousel items one at a time, and to announce the changes that occur in display.
 
 ## Position slides
```
Comment (yatil) [inline source/carousels/functionality.html.erb.md]: Well, it is the basic set up and people will have a hard time to understand the following portions if they don’t know how the slides are positioned and hidden…

**Pull request merged**